tests/common.py

```diff
@@ -47,6 +47,7 @@
 
 detect_leaks = os.environ.get("PITIVI_TEST_DETECT_LEAKS", "0") not in ("0", "")
 os.environ["PITIVI_USER_CACHE_DIR"] = tempfile.mkdtemp(suffix="pitiviTestsuite")
+os.environ["DISPLAY"] = "99.0"
```

Comment:
How did you come up with this value?

Reply:
In `ptv_testsuite.py`, I added code to print every `environment variable` (both key and value along with type of value).
It printed  `99.0` and type `string`. 
So I used the same value in `tests/common.py`.

Reply:
Great, thanks! :)

Reply:
This will work only inside flatpak.

Reply:
why?

Reply:
I used this command to see if it works:
```
$ nose2 tests.test_render.TestRender.test_rendering_with_scale tests.test_render.TestRender.test_rendering_with_scale -v
```

And it works, but only in the dev env. As thiblahute said, `99.0` seems to be something specific to flatpak. If I use it outside of the dev env I get an error:
```
$ DISPLAY="99.0" xterm
xterm: Xt error: Can't open display: 99.0
```

I found here the spec for `DISPLAY`, but it only confused me:
https://www.x.org/releases/X11R7.7/doc/man/man7/X.7.xhtml#heading5

I noticed that if I `del os.environ["DISPLAY"]` in common.py, it has the same effect as setting it to "99.0". Strangely, if I enter the sandbox by running `ptvenv` and unset `DISPLAY` before running the tests fails:
```
$ echo $DISPLAY 
:99.0
$ unset DISPLAY 
$ nose2 tests.test_render.TestRender.test_rendering_with_scale
(nose2:14): Gdk-WARNING **: cannot open display: 
```

So I should see how this works out of the dev env..

Reply:
Hrm, `Unable to init server: Could not connect: Connection refused`. 

We need to look into what exactly causes those windows to appear.

Reply:
Added some comments in https://phabricator.freedesktop.org/T7935 about next steps.

Reply:
The bug has been moved to https://gitlab.gnome.org/GNOME/pitivi/issues/2159

We could use this workaround, as long as the user is in the dev env, see https://gitlab.gnome.org/GNOME/pitivi/blob/master/bin/pitivi.in#L54

It would be great if you can make time to update the commit, otherwise let's close this PR.